lib/AST/ASTContext.cpp

```diff
@@ -22,11 +22,13 @@ extractLibraries(const std::vector<std::string> &LibrariesAddressInfo) {
     std::string Buff;
     while (SS >> Buff) {
       std::string LibName, Address;
-      size_t I = 0;
-      for (; Buff.at(I) != ':'; ++I)
-        LibName += Buff.at(I);
-      for (++I; I < Buff.size(); ++I)
+      int I = int(Buff.size()) - 1;
```

Comment:
You should use `size_t` not `int` when you want to retrieve the value of string length.

Reply:
fixed